src/DataSet/DataSet.php

```diff
@@ -17,49 +21,49 @@ class DataSet {
 	/**
 	 * Connection object.
 	 *
-	 * @var Zumba\PHPUnit\Extensions\Mongo\Client\Connector
+	 * @var Connector
 	 */
 	protected $connection;
 
-	/**
-	 * Constructor.
-	 *
-	 * @param Zumba\PHPUnit\Extensions\Mongo\Client\Connector
```

Comment:
@neok For auto-completion in many IDEs, it's better to leave the full namespace rather than an abbreviated one here (this goes for all the other changes as well).